instrumentation/http/src/main/java/brave/http/HttpHandler.java

```diff
@@ -15,6 +15,9 @@
 
 import brave.Span;
 import brave.internal.Nullable;
+import brave.internal.Platform;
+
+import static brave.internal.Throwables.propagateIfFatal;
 
 abstract class HttpHandler {
```

Comment:
note: changes to this file are mainly to it almost identical to RpcHandler in #999